test/psp.cpp

```diff
@@ -189,6 +189,19 @@ TEST(TABLE, simplest_test)
     tbl.reserve(5);
 }
 
+TEST(GNODE, implicit_pkey)
+{
+    /**
+     * NOTE This test should abort
```

Comment:
Can you change NOTE to TODO. That way various editors will highlight it as an action item.